### DISCUSSION TITLE
lotus-miner run - DiscoverNAT error:no NAT found"

### DISCUSSION THREAD

**Discussion by whhpc19891120** [filecoin-project/lotus#7320] [Storage Provider] [answered]
what can i do , please! 

{"level":"info","ts":"2021-09-01T15:33:39.104+0800","logger":"basichost","caller":"basic/natmgr.go:91","msg":"DiscoverNAT error:no NAT found"}

**Comment by de-authority** [comment, marked as answer]
It's not an error. it's info level log. your miner is ok.
